test/integration/crdvalidation.go

```diff
@@ -1,17 +1,19 @@
 package integration
 
 import (
-	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
 	"github.com/RHsyseng/operator-utils/pkg/validation"
 	"github.com/ghodss/yaml"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 	brokerv2alpha1 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha1"
+	brokerv2alpha2 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha2"
+
 	"io/ioutil"
 	"strings"
 )
 
 var crdTypeMap = map[string]interface{}{
-	"broker_v2alpha1_activemqartemis_crd.yaml":        &brokerv2alpha1.ActiveMQArtemis{},
+	"broker_v2alpha1_activemqartemis_crd.yaml":        &brokerv2alpha2.ActiveMQArtemis{},
 	"broker_v2alpha1_activemqartemisaddress_crd.yaml": &brokerv2alpha1.ActiveMQArtemisAddress{},
```

Comment:
if there are part of v2alpha2, shouldn't we update the filename of the CRD to match that? Should we be using one CRD from v1 and another from v2 or should they all be available and used from v2?

Reply:
Agreed, we need to change CRD files, will do the multiple CRD version validation another PR, above change temporarily resolve the compilation error.